src/Component/Facet/Range.php

```diff
@@ -48,6 +48,42 @@ public function getMinCount(): ?int
         return $this->getOption('mincount');
     }
 
+    /**
+     * @param string $tag
+     *
+     * @return \Solarium\Core\Configurable
+     */
+    public function setTag($tag)
+    {
+        return $this->setOption('tag', $tag);
+    }
+
+    /**
+     * @return string
+     */
+    public function getTag()
+    {
+        return $this->getOption('tag');
+    }
+
+    /**
+     * @param string|array $pivot
+     *
+     * @return \Solarium\Core\Configurable
+     */
+    public function setPivot($pivot)
```

Comment:
Don't these additions apply to JsonRange as well and should therefore be moved to the AbstractRange base class?

Reply:
moved to the abstract range...